util/emailUtil.js

```diff
@@ -0,0 +1,53 @@
+const nodemailer = require('nodemailer');
+
+const AppConfig = require('../config').AppConfig;
+
+// CC recipients are ones included in the email, but not intended directly for them 
+// in this case this will be the (super)admins of the company
+/**
+ * 
+ * @param {string} subject 
+ * @param {string} content 
+ * @param {string[]} recipients 
+ * @param {string[]} ccRecipients 
+ */
```

Comment:
I am thinking we should add it to all our methods. It will make code maintenance easier down the road. I don't think we will be able to reminisce what the code was about 4,5 months down the road.

Reply:
I agree. @srkacha @djuricic1 what's your opinion?

Reply:
If no further input is provided, I will leave it as is.